generators/wp/templates/wordpress-alpine/phpunit.xml

```diff
@@ -6,16 +6,23 @@
 	convertNoticesToExceptions="true"
 	convertWarningsToExceptions="true"
 >
+	<php>
+		<!--// These can be override via env variable //-->
+		<env name="VIP_GO_ENV" value="true" />
+		<env name="PMC_PHPUNIT_BOOTSTRAP" value="/var/www/html/wp-content/plugins/pmc-plugins/pmc-unit-test/bootstrap.php" />
```

Comment:
We should output the var for this path inside this value.

Reply:
That var already output as env file should override this.  Was added here as a backup default.  I will update as suggest, no harm here.  We really don't need this here since we have env file.  Was added here to sync up with the manual setup documentation.